contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.28;
+pragma solidity 0.8.24;
```

Comment:
Why are we decreasing the solidity version?

Reply:
i was told to use `0.8.24` by Rens from the core team. not entirely sure why, but it sounds like it has to do with deployments